### PR TITLE
Update Javascript Documentation “get-country-list/index”

### PR DESCRIPTION
**Pull request by mohsen-deriv**
Automatically generated by Netlify CMS

### CODE REVIEW

File: docs/languages/javascript/get-country-list/index.md [hunk 1]
```diff
@@ -2,6 +2,121 @@
...
+```
+
+We this call, you'll get useful information for the supported countries, such as:
+
+- `2-letter` code for each country
+- `Identity` service providers for each country
+- Country Tax Identifier Format (`tin_format`)
+- etc
+
+This can be useful on account creation forms, in which you need to ask your users to provide validated information about their identity base on their country of residence.
```
Comment (hubert-deriv) [inline docs/languages/javascript/get-country-list/index.md:128]: I am getting a little confused reading this sentence, I rephrased/rewrote the last part of the sentence. Is this correct? `... to provide validated information about their identity base, depending on their country of residence.`

File: docs/languages/javascript/get-country-list/index.md [hunk 2]
```diff
@@ -2,6 +2,121 @@
...
+
+:::caution
+For address, tax ID validations please the provided `tin_format` for the country.
```
Comment (sanjam-deriv) [inline docs/languages/javascript/get-country-list/index.md:111]: Not sure what this line means, maybe it is a grammatical error

**Review by sanjam-deriv — CHANGES_REQUESTED** [review]

File: docs/languages/javascript/get-country-list/index.md (hunk 2 above)
Comment (hubert-deriv) [inline docs/languages/javascript/get-country-list/index.md]: Also have trouble understanding this sentence, is this perhaps a better way of writing it?`For address and tax ID validations, please use the provided 'tin_format' for the country.`

File: docs/languages/javascript/get-country-list/index.md [hunk 3]
```diff
@@ -2,6 +2,121 @@
...
+:::
+
+The country of your users is important for our next steps, since with it you can determine which assets and features the client can use.
```
Comment (hubert-deriv) [inline docs/languages/javascript/get-country-list/index.md:134]: `The country of your users is important for our next steps. With it, you can determine which assets and features the client can use.`

File: docs/languages/javascript/get-country-list/index.md [hunk 4]
```diff
@@ -2,6 +2,121 @@
...
+  - javascript
+description: How to get list of countries in deriv api?
 ---
 
-For setting up the websocket connection....
+:::caution
+You can learn more about countries [here](/docs/core-concepts/residence-list)
+:::
+
+To get countries list, please update the `open` event listener like so:
```
Comment (hubert-deriv) [inline docs/languages/javascript/get-country-list/index.md:18]: change `countries list` to `the countries list`

File: docs/languages/javascript/get-country-list/index.md (hunk 2 above)
Comment (mohsen-deriv) [reply docs/languages/javascript/get-country-list/index.md]: yeah, the `use` is missing, lol, fixed thanks man :pray:

**Review by hubert-deriv — CHANGES_REQUESTED** [review]

File: docs/languages/javascript/get-country-list/index.md [hunk 5]
```diff
@@ -2,6 +2,121 @@
...
+      "phone_idd": "355",
+      "text": "Albania",
+      "tin_format": ["^[A-Ta-t0-9]\\d{8}[A-Wa-w]$"],
+      "value": "al"
+    }
+  ]
+}
+```
+
+We this call, you'll get useful information for the supported countries, such as:
```
[the remark this answers is not in the document]
Comment (mohsen-deriv) [reply docs/languages/javascript/get-country-list/index.md:121]: maaan, it's hard to write things with human language :(
yeah, will fix it. thanks thanks :pray:

File: docs/languages/javascript/get-country-list/index.md (hunk 1 above)
Comment (mohsen-deriv) [reply docs/languages/javascript/get-country-list/index.md:128]: awesome, thanks

File: docs/languages/javascript/get-country-list/index.md (hunk 2 above)
Comment (mohsen-deriv) [reply docs/languages/javascript/get-country-list/index.md]: sure sure

File: docs/languages/javascript/get-country-list/index.md (hunk 4 above)
Comment (mohsen-deriv) [reply docs/languages/javascript/get-country-list/index.md]: yeah cool, thanks :pray:

File: docs/languages/javascript/get-country-list/index.md [hunk 6]
```diff
@@ -2,6 +2,192 @@
 title: Get list of countries
 sidebar_label: Get list of countries
 sidebar_position: 2
+tags:
+  - country_list
+  - javascript
+keywords:
+  - country_list
+  - javascript
+description: How to get list of countries in deriv api?
```
Comment (matin-deriv) [inline docs/languages/javascript/get-country-list/index.md:11]:
```suggestion
description: How to get the list of countries in Deriv api?
```

**Review by hubert-deriv — APPROVED** [review]

**Review by matin-deriv — APPROVED** [review]

**Review by sanjam-deriv — APPROVED** [review]

**Pull request merged**